changelog/parse_changelogs.py

```diff
@@ -0,0 +1,37 @@
+#!/bin/python3
+
+import os
+import re
+
+
+def combine_sections_in_folder(folder_path):
```

Comment:
most of this is generated by AI, I only made small changes to make it work correctly